packages/terra-site/src/examples/paginator/SearchPaginatorExample.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Dialog from 'terra-dialog';
+import Paginator from 'terra-paginator/lib/Paginator';
+
+const totalCount = 450;
+
+const fillArray = (value, len) => {
+  const arr = [];
+  for (let i = 0; i < len; i += 1) {
+    arr.push(<p key={Math.floor(Math.random() * Math.floor(100000))}>{value}</p>);
+  }
+  return arr;
+};
+
+const buildPage = () => {
+  let fullContent = [];
+  const content = ('Lorem ipsum dolor sit amet, consectetur adipiscing elit. ' +
+    'Fusce porttitor ullamcorper nisi, vel tincidunt dui pharetra vel. ' +
+    'Morbi eu rutrum nibh, sit amet placerat libero. Integer vel dapibus nibh. ' +
+    'Donec tempor mi vitae lorem congue, ut ultrices metus feugiat. Sed non commodo felis. ' +
+    'Aliquam eget maximus dui, ut rhoncus augue.');
+
+  fullContent = fillArray(content, 10);
+
+  return (
+    fullContent
+  );
+};
+
+class SearchPaginatorExample extends React.Component {
```

Comment:
Does this still need to be SearchPaginator?

Reply:
https://github.com/cerner/terra-core/pull/1445/commits/7827f90bf4179514cd22d77a2b46b608bdc8c660